Tasks/nba_cup.rb

```diff
@@ -0,0 +1,52 @@
+#You are given a string with results of NBA teams separated by commas e.g:
+
+result_sheet = "Los Angeles Clippers 104.2 Dallas Mavericks 88,New York Knicks 101 Atlanta Hawks 112,Indiana Pacers 103 Memphis Grizzlies 112,"\
+"Los Angeles Lakers 111 Minnesota Timberwolves 112,Phoenix Suns 95 Dallas Mavericks 111,Portland Trail Blazers 112 New Orleans Pelicans 94,"\
+"Sacramento Kings 104 Los Angeles Clippers 111,Houston Rockets 85 Denver Nuggets 105,Memphis Grizzlies 76 Cleveland Cavaliers 106,"\
+"Milwaukee Bucks 97 New York Knicks 122,Oklahoma City Thunder 112 San Antonio Spurs 106,Boston Celtics 112 Philadelphia 76ers 95"
+
+#Given a team (parameter : to_find) e.g:
+to_find = "Los Angeles Clippers"
+
+# nba_cup(result_sheet,to_find)
+
+#returns "Team Name:W=nb of wins;D=nb of draws;L=nb of losses;Scored=nb;Conceded=nb;Points=nb"
+
+#If a team name can't be found in the given string 
+#returns "Team Name:This team didn't play!".
+
+#If a score is a float number will 
+#returns "Error(float number):the concerned string"
+
+#If the team name "" returns ""
+
+def nba_cup(result_sheet, to_find)
```

Comment:
this method could be split into logical methods. Each responsible for its own part. 1. Parsing Input 2. Processing data. 3. Presenting output.